packs/lacework/dashboards/lacework-compliance-violations.json

```diff
@@ -0,0 +1,229 @@
+{
+  "name": "Lacework Compliance Violations",
+  "description": null,
```

Comment:
same comments here related to description fields and permissions

Reply:
Done!